adaptive_alerting_detector_build/cli.py

```diff
@@ -1,228 +1,121 @@
-"""Adaptive Alerting Detectors CLI Tool
+"""Adaptive Alerting Detector Build
 
-Creates, re-trains or deletes Adaptive Alerting Constant Threshold detectors, based on the provided
+Creates, re-trains, and deletes Adaptive Alerting detectors, based on the provided
 JSON metrics configuration file.
 
 Usage:
-    adaptive-alerting build --datasource=url --aaservice=url --username=name [--weakmultiplier=number] [--strongmultiplier=number] <json_config>...
-    adaptive-alerting delete --aaservice=url <json_config>...
+    adaptive-alerting build <json_config_file>...
+    adaptive-alerting delete <json_config_file>...
     adaptive-alerting -h | --help
 
 Commands:
-    build       builds new detectors or re-trains existing detectors for the metric
-    delete      deletes detectors for the metric
+    build       builds new detectors for each metric
+    train       trains existing detectors for each metric
+    delete      deletes detectors for each metric
 
 Options:
-    --datasource=url        URL for the datasource
-    --aaservice=url         URL for the Adaptive Alerting Model Service
-    --username=name         Alias of the user that will be stored with the detector
-    --weakmultiplier=num    The multiplication factor for weak anomalies [default: 3]
-    --strongmultiplier=num  The multiplication factor for strong anomalies [default: 5]
-    json_config             One or more configuration files containing the metric configuration
+    json_config_file          One or more configuration files containing the metric configuration
     -h --help               Show this screen
 
 Examples:
-    adaptive-alerting build --datasource=https://graphite.company.com --aaservice=https://adaptive-alerting.company.com/modelservice --username=msmith service-name.json
+    adaptive-alerting build metrics.json
 
-    adaptive-alerting build --datasource=https://graphite.company.com --aaservice=https://adaptive-alerting.company.com/modelservice --username=msmith service-name1.json service-name2.json
+    adaptive-alerting build metrics.json
```

Comment:
This is a duplicate of the line above. Maybe keep it and show how you can provide more than one json file.